app/src/main/java/org/sagebionetworks/research/mpower/logging/LoggingViewModelFactory.java

```diff
@@ -0,0 +1,52 @@
+package org.sagebionetworks.research.mpower.logging;
+
+import android.arch.lifecycle.ViewModel;
+import android.arch.lifecycle.ViewModelProvider;
+import android.support.annotation.NonNull;
+
+import org.sagebionetworks.bridge.android.manager.ActivityManager;
+import org.sagebionetworks.research.mpower.TaskLauncher;
+
+import javax.inject.Inject;
+
+/**
+ * A ViewModelFactory allows us to instantiate ViewModels using creation parameters and/or constructor arguments.
+ * LoggingViewModel's create method does not take parameters, but it does take a constructor argument.
+ */
+public class LoggingViewModelFactory {

```

Comment:
Does "Logging" mean the same thing as the iOS equivalent, for data tracking?

Reply:
Oops, not sure why I thought the tab was named `Logging` instead of `Tracking`